lego/apps/events/tasks.py

```diff
@@ -145,6 +145,13 @@ def async_unregister(self, registration_id, logger_context=None):
             exception=e,
             registration_id=registration.id,
         )
+        registration.status = constants.FAILURE_UNREGISTER
+        registration.save()
+        notify_user_registration(
+            constants.SOCKET_UNREGISTRATION_FAILURE,
+            registration,
+            error_message="Avregistrering feilet",
```

Comment:
Wouldn't hurt to add why in the error message as well hehe

Reply:
Fair, just copied from the other exception